src/ecs/resources/input.rs

```diff
@@ -22,14 +22,52 @@ impl<'a> Iterator for PressedKeysIterator<'a> {
     }
 }
 
+/// Iterator that iterates through all currently pressed down mouse buttons
+pub struct PressedMouseButtonsIterator<'a> {
```

Comment:
Not true, that is a breaking change, however we are following a specific type of semver for sub 1.0.0 projects, this means that changes like these don't need a major bump.